### ISSUE TITLE
Conflict between abbreviated and regular versions of "day" and "days"

### ISSUE THREAD

**Issue by damienrambout** [mattt/FormatterKit#232]
Hi,

I noticed that `.strings` files only reference translations for `day` and `days` keys. There's no reference to their abbreviated form, except for comments (e.g. Day Unit (Singular, Abbreviated)).

This issue was already addressed in #144 and it was partially fixed for the singular abbreviated value (adding `d` localization key). Partially only because `.strings` were not updated to reference this `d` localization key. Which leads to incoherent behaviors in languages such as German were *1 day* is translated into *1 T.*, meaning there's not regular version for this value in German.
About the plural form, `days` is still referenced twice in the code (for regular **and** abbreviated values). We should add a `ds` localization key as well.

All `.strings` files should be updated to add the abbreviated values instead of reusing `day` and `days` for abbreviated values.

**TL;DR**:
`.strings` file should contain the following keys `day`, `days`, `d` and `ds` (instead of only `day` and `days`). Translations should be added for the abbreviated versions.

and

```objc
NSLocalizedStringFromTableInBundle(@"days", @"FormatterKit", [NSBundle bundleForClass:[self class]],  @"Day Unit (Plural, Abbreviated)");
```
should be
```objc
NSLocalizedStringFromTableInBundle(@"ds", @"FormatterKit", [NSBundle bundleForClass:[self class]],  @"Day Unit (Plural, Abbreviated)");
```

I'll make a pull-request in a near future.

**Comment by yas375** [comment]
Thanks for the fix!
